### DISCUSSION TITLE
Booking system in Statamic.

### DISCUSSION THREAD

**Discussion by chethanidaksh** [statamic/cms#7405] [General]
### Can we implement a booking system in Statamic 
-> Users can register themselves and book their orders via a form
-> that form will be sent to the admin mail address 
			
Is this possible in Statamic? if so show me some way. thanks.

**Comment by chethanidaksh** [comment]
Doing some research I got this https://v1.statamic.com/learn/advanced-features/membership
But this is Statamic version 1.

**Reply by jackmcdade** [reply]
v3's verison here: https://statamic.dev/tags/user-register_form
